src/assets/src/validation.ts

```diff
@@ -86,8 +86,6 @@ export function validateString (value: string, schema: StringSchema<string>, sho
         const maxLimit = getMaxLimit(schema.describe());
         if (showRemaining && maxLimit) {
             messages.push(createRemainingCharsMessage({value: transformedValue, max: maxLimit}));
-        } else {
-            console.warn('validateString called with showReamining but schema does not include a maxiumum.');
```

Comment:
What is the user see from the home screen/UI when this scenario occurs?

Reply:
Nothing, this just shows up a bunch in the console log.

Reply:
If `showRemaining` is `false` (like with the uniqname input on the queue management screen), then there just isn't a remaining message, like there is for the larger fields.

We could add a separate check for the scenario where `showRemaining` is `true` and `maxLimit` is `undefined`, but it doesn't seem super important. See the issue for more details.

Reply:
I did test his use case last time, as you said user might not care unless the UI doesn't notify him in case of error. Which seems to be not the case here.